### PR TITLE
Bumps CAPC to new commit with bug fix

### PR DESCRIPTION
**Pull request by wanyufe**
*Issue #, if available:*
CAPC has bug fix of adding zoneID when retrieving serviceOffering and diskOffering.
*Description of changes:*
Refer to new CAPC repo commmit hash

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: UPSTREAM_PROJECTS.yaml [hunk 1]
```diff
@@ -130,7 +130,7 @@ projects:
           - tag: v1.2.0
       - name: cluster-api-provider-cloudstack
         versions:
-          - tag: v0.4.9-rc3
+          - commit: cefafbebf46668a36a690e076d25a35f6d24ff3f
```
Comment (jiayiwang7) [inline UPSTREAM_PROJECTS.yaml:133]: can we use a git tag instead of commit?

Comment (pmotyka) [reply UPSTREAM_PROJECTS.yaml:133]: for this fix we may have to go with commit hash unfortunately

**Comment by lewisdiamond** [comment]
This breaks the cloudstack provider, the commit hash cannot be used directly. Can a tag be created instead?

**Pull request merged**